utils/generate_types/index.js

```diff
@@ -239,6 +260,12 @@ function parentClass(classDesc) {
 
 function writeComment(comment, indent = '') {
   const parts = [];
+  
+  comment = comment.replace(/\[`([^`]+)`\]\(#([^\)]+)\)/g, '[`$1`](https://github.com/microsoft/playwright/blob/master/docs/api.md#$2)');
```

Comment:
Should we have a shared place for all these magic md regexes scattered around the files?

Reply:
This code is volatile, I don't think sharing these makes sense.